packages/imitation_game/last_results.json

```diff
@@ -0,0 +1,14 @@
+{
+  "smiley": {
+    "flutter": {
+      "startupTime": 0.691717,
+      "adb_memory_total": 101928.0
```

Comment:
man this is brutal. Where's the code that populated this measurement? I assume this is a strict delta measurement between before Flutter started and after Flutter started? Did we call the gc before each measurement? Can we break it down into memory categories to make this more actionable?

Reply:
It's just the smiley application.  If you run `adb shell dumpsys meminfo $BUNDLE_NAME` after launching that app on an android device you can see the breakdown.  Figuring out how to get the release build working may help a bit.

Reply:
Also, I don't think it would be fair to call "GC" before measuring since that isn't how people are suppose to write their apps.

Reply:
I ran this again today and with today's flutter, flutter wins.  I'll add the flutter version to the results...

Reply:
(Done, added the flutter version in the output)

Reply:
ah good to know you're getting it just for the process. I think GC before measuring is fairly standard. It's not fair if you only do it in Flutter. I'd expect GC to be called for both the Android and Flutter trials. Otherwise it'd be "non-hermetic" and repeat runs won't be consistent.